File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/condicionais/q01.py
def main():
    #Input

    num1 = int(input("Enter a number: "))
    num2 = int(input("Enter a second number: "))
    num3 = int(input("Enter a third number: "))

    #Processing
    count = 0
    if num1 == num2:
        count += 1

        if num1 == num3:
            count += 1

            if num2 == num3:
                count += 1

    #Output
    print(f"The total igual number is to: {count}")

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-02/q01.js
import { question } from "readline-sync";

function main(){
    //Input
    const num1 = Number(question("Enter a number: ")) 
    const num2 = Number(quesiton("Enter a second number: "))
    const num3 = Number(question("Enter a third number: "))

    //Processing

    const count = 0;   
    
    if (num1 == num2){
        count += 1
    }
    
    else if (num1 == num3){
        count += 1
    }

    else if (num2 == num3){
        count += 1
    }

    //Output
    console.log(`A quantidade de numeros iguais e: ${count}`)
}
main()